ts/features/payments/common/components/PaymentCard.tsx

```diff
@@ -24,6 +24,7 @@ export type PaymentCardProps = {
   holderPhone?: string;
   holderEmail?: string;
   isExpired?: boolean;
+  pspBusinessName?: string;
```

Comment:
Why this prop here? We do not use it inside the `PaymentCard` component. I'd suggest to move it elsewhere

Reply:
Addressed into 0b567a71d1844da20f4429ad4ea5b814d2712a2a